package.json

```diff
@@ -75,6 +75,7 @@
     "externalip": "^1.0.2",
     "github-release-notes": "^0.16.0",
     "hdkey": "^0.8.0",
+    "ip": "^1.1.5",
```

Comment:
@kipliklotrika does package-lock.json needs to be updated and commited as well?

Reply:
Nope, since it's already used by other package and is already there.